packages/gatsby/src/services/create-pages.ts

```diff
@@ -38,31 +41,31 @@ export async function createPages({
   )
   activity.end()
 
-  // reporter.info(`Checking for deleted pages`)
+  reporter.info(`Checking for deleted pages`)
```

Comment:
The reason I had it was because in some cases it was taking a few seconds, and it was confusing working out where this was happening. It should be super quick now though, so I guess it's not needed at that level. I'll change to verbose